docs/plugin-development-guide/implementation.rst

```diff
@@ -86,54 +54,23 @@ PHPSpec, PHPUnit, or any other unit testing tool):
 
     <?php
 
-    // src/Entity/ProductVariantInterface.php
+    // src/Model/ProductVariantInterface.php
 
     declare(strict_types=1);
 
     namespace IronMan\SyliusProductOnDemandPlugin\Entity;
 
-    use Sylius\Component\Core\Model\ProductVariantInterface as BaseProductVariantInterface;
-
-    interface ProductVariantInterface extends BaseProductVariantInterface
+    interface ProductVariantInterface
     {
         public function setAvailableOnDemand(bool $availableOnDemand): void;
 
         public function isAvailableOnDemand(): bool;
     }
 
-Of course you need to remember about entity mapping customization as well:
-
-.. code-block:: yaml
-
-    # src/Resources/config/doctrine/ProductVariant.orm.yml
-
-    IronMan\SyliusProductOnDemandPlugin\Entity\ProductVariant:
-        type: entity
-        table: sylius_product_variant
-        fields:
-            availableOnDemand:
-                type: boolean
-
-Then our new entity should be configured as a resource model:
-
-.. code-block:: yaml
-
-    # src/Resources/config/config.yml
-
-    sylius_product:
```

Comment:
What do you think about recommending this configuration in the test folder, so the entity will be checked if wired up properly (and preferably will be a good introduction for testing)